Production/govuk_ios/Helpers/Redactor/Redactables.swift

```diff
@@ -0,0 +1,18 @@
+import Foundation
+
+enum Redactables {
+    case email
+    case niNumber
+    case postCode
+
+    var fetchRedactable: Redactable {
```

Comment:
I'd be inclined to rename this to 'type' or 'redactableType' (or whatever you decide the enum is named, + Type).  'fetch' implies a function to me (as well as some kind of storage), and this is just a variable